frontend/src/components/admin/assessment-creation/PreviewAssessmentModal.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from "react";
+import { Button } from "@chakra-ui/react";
+
+import { EditOutlineIcon } from "../../../assets/icons";
+import AssessmentContext from "../../../contexts/AssessmentContext";
+import AssessmentExperience from "../../student/AssessmentExperience";
+
+const PreviewAssessmentModal = (): React.ReactElement => {
```

Comment:
Similar comment to the other PR, I feel like maybe `PreviewAssessmentPage` might be more appropriate here given that this is different from our other modal components